sys/net/gnrc/network_layer/ipv6/nib/nib.c

```diff
@@ -1458,17 +1458,18 @@ static void _handle_rtr_timeout(_nib_dr_entry_t *router)
 {
     if ((router->next_hop != NULL) && (router->next_hop->mode & _DRL)) {
         _nib_offl_entry_t *route = NULL;
-        unsigned iface = _nib_onl_get_if(router->next_hop);
-        ipv6_addr_t addr = router->next_hop->ipv6;
+        _nib_onl_entry_t *next_hop = router->next_hop;
 
         _nib_drl_remove(router);
-        /* also remove all routes to that router */
+        /** The Router Lifetime applies only to
```

Comment:
```suggestion
        /* The Router Lifetime applies only to
```

`/**` are doxygen comments so not quite sure why `/*` would not suffice.